### DISCUSSION TITLE
Webpack 5.96 Breaking Change

### DISCUSSION THREAD

**Discussion by josheppinette** [webpack-contrib/webpack-bundle-analyzer#647] [Q&A]
**Result**
The gzip and parsed treemap sizes are missing and the following error is being thrown in the console:

```
Error parsing bundle asset "/Users/<correct-path>/root.bc550a233bc7d9561c2f.js": 'import' and 'export' may appear only with 'sourceType: module' (1:0)

No bundles were parsed. Analyzer will show only original module sizes from stats file.
```

**Cause**
Upgrading Webpack 5.95.X to 5.96.X.

**Time Since Regression**
Webpack v5.96 was released on October 31.

**Reproduction**
- https://github.com/jteppinette/webpack-bundle-analyzer-regression-reproduction

**Comment by valscion** [comment]
Can you create a minimal reproduction case? And maybe look around if you could find another issue for this same topic. I think I've seen this reported somewhere else earlier but can't remember where

**Reply by josheppinette** [reply]
Here is the [reproduction](https://github.com/jteppinette/webpack-bundle-analyzer-regression-reproduction).

I have seen many issues related to this (i.e. same result), but with a different cause.

**Reply by josheppinette** [reply]
@valscion any thoughts? I can dig into this more, but a slight point in the right direction would be helpful.

**Reply by valscion** [reply]
Thanks for the reproduction!

Ah so it's most likely `output.module = true` that creates a bundle which the acorn parser even fails to start parsing.

Here's the function which probably would need attention to figure this out:

https://github.com/webpack-contrib/webpack-bundle-analyzer/blob/f8d4711d7670ffa9fae11cd34787cc45c538f9bb/src/parseUtils.js#L9-L17

My hunch is that the `sourceType: 'script'` is what causes this breakage.

I don't know if changing the sourceType would break parsing other, more commonly used output formats, though. The tests should fail if so, though.
